import-service/src/handlers/importFileParser.js

```diff
@@ -7,35 +7,51 @@ const { IMPORT_BUCKET_NAME, IMPORT_CATALOG_NAME, PARSED_CATALOG_NAME } = process
 
 const importFileParser = async (event) => {
```

Comment:
Could you add log - console.log('Function execution started with event: ', JSON.stringify(event))